apps/antalmanac/src/routes/Home.tsx

```diff
@@ -1,55 +1,79 @@
 import DateFnsUtils from '@date-io/date-fns';
-import { Box, CssBaseline, useMediaQuery, useTheme } from '@material-ui/core';
 import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { CssBaseline, useMediaQuery, useTheme } from '@mui/material';
+import { Stack } from '@mui/material';
 import Split from 'react-split';
 
 import Calendar from '$components/Calendar/CalendarRoot';
 import Header from '$components/Header';
 import NotificationSnackbar from '$components/NotificationSnackbar';
 import PatchNotes from '$components/PatchNotes';
-import SharedTabs from '$components/SharedRoot';
+import ScheduleManagement from '$components/SharedRoot';
 import { Tutorial } from '$components/Tutorial';
 
-export default function Home() {
-    const isMobileScreen = useMediaQuery('(max-width: 750px)');
+function MobileHome() {
+    return (
+        <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <CssBaseline />
+
+            <PatchNotes />
+
+            <Stack component="main" height="100dvh">
+                <Header />
+                <ScheduleManagement />
+            </Stack>
+
+            <NotificationSnackbar />
+        </MuiPickersUtilsProvider>
+    );
+}
+
+function DesktopHome() {
     const theme = useTheme();
 
     return (
         <MuiPickersUtilsProvider utils={DateFnsUtils}>
             <CssBaseline />
+
             <PatchNotes />
+
             <Header />
 
-            {isMobileScreen ? (
-                <SharedTabs mobile={true} />
-            ) : (
-                <>
-                    <Split
-                        sizes={[45, 55]}
-                        minSize={100}
-                        expandToMin={false}
-                        gutterSize={10}
-                        gutterAlign="center"
-                        snapOffset={30}
-                        dragInterval={1}
-                        direction="horizontal"
-                        cursor="col-resize"
-                        style={{ display: 'flex' }}
-                        gutterStyle={() => ({
-                            backgroundColor: theme.palette.primary.main,
-                            width: '10px',
-                        })}
-                    >
-                        <Box>
-                            <Calendar isMobile={false} />
-                        </Box>
-
-                        <SharedTabs mobile={false} />
-                    </Split>
-                    <Tutorial />
-                </>
-            )}
+            <Split
+                sizes={[45, 55]}
+                minSize={100}
+                expandToMin={false}
+                gutterSize={10}
+                gutterAlign="center"
+                snapOffset={30}
+                dragInterval={1}
+                direction="horizontal"
+                cursor="col-resize"
+                style={{ display: 'flex', flexGrow: 1, height: '100dvh' }}
```

Comment:
```suggestion
                style={{ display: 'flex', flexGrow: 1, height: 'calc(100dvh - 50px)' }}
```
issue: we're not accounting for the header height

<img width="1313" alt="Screenshot 2024-09-30 at 2 53 06 PM" src="https://github.com/user-attachments/assets/4ca921d1-61a6-44ba-96ac-37505d711c14">

Reply:
Suggestion is wrong. Calc is terrible because we're guessing what the height of the header is, instead of relying on the browser's exact knowledge. Solution is to wrap the header and content with a different container that is 100dvh, and then ask the content to fill up the rest of the height after the header.